documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+as well as
+[Openverse’s Code of Conduct](https://github.com/WordPress/openverse/blob/main/CODE_OF_CONDUCT.md).
+Please note that the consent decision-making document linked above also includes
+specific provisions for opting out of a decision discussion you do not wish to
+or cannot participate in.
+
+## Current round
+
+The discussion is currently in the {Question, Reaction, Revision, Objection
+raising, Objection revision} round.
+```
+
+## Round descriptions
+
+Please keep in mind the following general guidelines that apply to all rounds:
+
+- Reactions may be shared at any of the "feedback" rounds (Question and
+  Objection).
+- Proposal authors are not obligated to respond to every question, reaction, or
+  objection that is raised, except for paramount objections. While there is an
+  expectation that the author will address outstanding questions, it isn't
+  necessary to respond to every non-paramount objection before moving on to
+  subsequent rounds or ratifying the proposal. It is, of course, in the best
+  interest of the project and the proposal that as much clarity as possible is
+  achieved.
+- Authors may table a proposal at any time at their discretion. If the proposal
+  is tabled, discussion should not be expected to continue, but may, if there is
+  something important being discussed.
+- Any round with a suggested length of time is subject to
+  [shortcutting as described below](#shortcutting-a-round) or to extension at
+  the discretion of the participants.
+
+### Question round
+
+This round lasts for 3 business days after the discussion starts but can be
+extended if questions reveal significant gaps in the proposal that must be
+understood before the proposal can be revised in the revision step. These
+questions are different from objections in that they’re not in the form of "we
+cannot do this because x part of the proposal will not work". Rather, they are
+clarifying questions in the form of:
+
+- "I don’t understand y in the proposal, can you clarify what is meant by this?"
+- "I think a gap exists in the proposed solution here [clarify where]. Can you
+  fill in the details in this area?"
+
+In short, the goal of the question round is for everyone participating to fully
+understand the proposal. Participants are urged that many objections can be
+restated as questions. This helps promote good faith between participants and
+allows the author to clarify things they might have missed in the initial draft
+of the proposal. This does _not_ mean that known paramount objections should be
+reserved by participants. If you know that a proposed solution is impossible or
+illegal, please do raise this concern as soon as possible.
+
+Note that the author does not have to respond to every question. However, they
+should keep them in mind during the revision round.
+
+#### Reactions
+
+This round also invites participants to share reactions, positive or negative,
```

Comment:
> This round ~also~ invites participants to share reactions, positive or negative,

Reply:
I think the "also" is a fair qualification here considering it is in a sub-heading of the question round :sweat_smile: Without the "also" it reads in context as though "Reactions" is the name of another round.

Reply:
>  it reads in context as though "Reactions" is the name of another round.

...which was how I read it, oops 😅.